system-int-test/test/cmr/system_int_test/search/collection_sorting_search_test.clj

```diff
@@ -379,3 +384,55 @@
+                                  (t/plus (t/now) (t/years 1)))
+        year-in-past (f/unparse (f/formatters :date-time)
+                                (t/minus (t/now) (t/years 1)))
+        coll1 (d/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection 1 {:EntryTitle "Dataset1"
+                                         :TemporalExtents [(data-umm-cmn/temporal-extent
+                                                            {:beginning-date-time "2010-01-01T12:00:00Z"
+                                                             :ending-date-time year-in-past})]}))
+        coll2 (d/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection 2 {:EntryTitle "Dataset2"
+                                         :TemporalExtents [(data-umm-cmn/temporal-extent
+                                                            {:beginning-date-time "2010-01-01T12:00:00Z"
+                                                             :ending-date-time year-in-future})]}))
+        coll3 (d/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection 3 {:EntryTitle "Dataset3"
+                                         :TemporalExtents [(data-umm-cmn/temporal-extent
+                                                            {:beginning-date-time "2010-01-01T12:00:00Z"
+                                                             :ending-date-time outside-near-current})]}))
+        coll4 (d/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection 4 {:EntryTitle "Dataset4"
+                                         :TemporalExtents [(data-umm-cmn/temporal-extent
+                                                            {:beginning-date-time "2010-01-01T12:00:00Z"
+                                                             :ending-date-time inside-near-current})]}))
+        coll5 (d/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection 5 {:EntryTitle "Dataset5"
+                                         :TemporalExtents [(data-umm-cmn/temporal-extent
+                                                            {:beginning-date-time "2010-01-01T12:00:00Z"
+                                                             :ending-date-time today})]}))
+        coll6 (d/ingest-umm-spec-collection
+               "PROV1"
+               (data-umm-c/collection 6 {:EntryTitle "Dataset6"
+                                         :TemporalExtents [(data-umm-cmn/temporal-extent
+                                                            {:beginning-date-time "2010-01-01T12:00:00Z"})]}))]
+    (index/wait-until-indexed)
+    (are [sort-key items]
+         (sort-order-correct? items sort-key)
+
+         ["-near-current" "entry-title"] [coll2 coll4 coll5 coll6 coll1 coll3]
+         ["near-current" "entry-title"] [coll1 coll3 coll2 coll4 coll5 coll6])))
```

Comment:
I don't understand the result...   So, coll1, coll2 and coll3, they should all have the same near-current value, which is false right? So, sorting of these three should only be based on entry-title -  coll1, coll2, coll3?
I think you might want to add tests that's only sorting on the near-current first to make sure it's working as expected before mixing it with other sorting parameters.   If I have c1: near-current=true, c2: near-current=true, c3: near-current-false, c4: near-current=false, what order do I expect to see when I search for them with sort key = near-current?

Reply:
One question per comment is easier to respond to.
1. No, coll2 is true.  coll1 and coll3 are false.
2. The current sorting is right.
3. If entry-title is consistent: sort-key= near-current, then the order is c3, c4, c1, c2 because the default sort order for near-current is :asc so false goes before true.  I haven't figured out how to easily reverse it.

Reply:
Thanks. After Chris approves your logic, please add the doc update for this new sorting parameter.